src/components/utility-components/input/SummaryLengthSlider.tsx

```diff
@@ -55,14 +57,23 @@ const SummaryLengthSlider = ({
       </div>
       <div>
         <input
-          type="text"
+          type="number"
           className={`w-100 mt-[6.25rem] h-20 w-80 rounded-full border-[1px] border-solid border-primary p-[.625rem] text-center font-[.875rem] text-primary ${
             Number(summaryLength) > 300 ? "visible" : "invisible"
           }`}
           value={customLength}
+          required
           placeholder="Enter number here"
           id="custom-summary-length"
           onChange={(event) => {
+            if (Number(event.target.value) < minLength) {
+              setCustomLength(`${minLength}`);
+              return;
+            }
+            if (Number(event.target.value) > maxLength) {
+              setCustomLength(`${maxLength}`);
+              return;
+            }
```

Comment:
may be should validate only when the summarize button is pressed?

noticing a bug here, the field starts blank but once any number is entered, it changes to 10.. and not able to backspace and reenter the value..

Reply:
Good point! Thanks for catching!